test/integration/rules/runner.js

```diff
@@ -29,7 +29,13 @@
 
 	var fixture = document.getElementById('fixture');
 	Object.keys(tests).forEach(function(ruleId) {
-		describe(ruleId, function() {
+		// don't run deprecated rules
+		var rule = axe._audit.rules.find(function(rule) {
+			return rule.id === ruleId;
+		});
+		var deprecated = rule.tags.indexOf('deprecated') !== -1;
```

Comment:
If `rules.find()` somehow doesn't match, this will throw, since `rule` will be `undefined`. Do we want to guard against this?

Reply:
I thought about it, but the entire test script assumes there is a rule with that id, otherwise the `runOnly` option for `axe.run` would also through. It's still probably not a bad idea though.

Reply:
Fair enough!